### PR TITLE
docs: api/grn_table: move grn_table_setoperation() reference to the header file

### PR DESCRIPTION
**Pull request by abetomo**
GitHub: GH-1817

Prepare to switch to documents automatically generated by Doxygen.

### CODE REVIEW

File: include/groonga/table.h [hunk 1]
```diff
@@ -748,6 +748,26 @@ grn_table_group_keys_parse(grn_ctx *ctx,
...
+/**
+ * \brief The result of the set operation of `table1` and `table2` according to
+ *        the specification of `op` is stored in `res`.
+ *
+ * \attention The table specified in `res` will be destructed.
+ *
+ * \param ctx The context object.
+ * \param table1 The table.
+ * \param table2 The table.
+ * \param res Specify `table1` or `table2`.
```
Comment (otegami) [inline include/groonga/table.h:760]: Can we specify the other table except for `table1` or `table2`, although the current use-cases are handling only table1 or table2?

Comment (abetomo) [reply include/groonga/table.h:760]: If it is not `table1` or `table2`, the error `GRN_INVALID_ARGUMENT` occurs.

Comment (otegami) [reply include/groonga/table.h:760]: Ahh, I see. I missed it, Thanks you!

https://github.com/groonga/groonga/blob/5fcf66bd731ae6e79806447f6582c5bfbba1849b/lib/db.c#L5211

**Review by otegami — APPROVED** [review]
LGTM :pray:

File: include/groonga/table.h [hunk 2]
```diff
@@ -748,6 +748,26 @@ grn_table_group_keys_parse(grn_ctx *ctx,
                            const char *raw_sort_keys,
                            int32_t raw_sort_keys_size,
                            uint32_t *n_keys);
+/**
+ * \brief The result of the set operation of `table1` and `table2` according to
+ *        the specification of `op` is stored in `res`.
```
Comment (kou) [inline include/groonga/table.h:753]:
```suggestion
 * \brief Execute the set operation of `table1` and `table2` according to
 *        the specification of `op` and store the result into `res`.
```

**Pull request merged**